test/fixtures/sub-process.js

```diff
@@ -4,7 +4,15 @@ var Insight = require('../../lib');
 var insight = new Insight({
 	packageName: 'yeoman',
 	packageVersion: '0.0.0',
-	trackingCode: 'GA-1234567-1'
+	trackingCode: 'GA-1234567-1',
+	config: {
```

Comment:
I'd rather not bind up this externally-observable change on a minor overhaul of testing patterns. Would you be willing to land the PR as-is and revisit `process` mocking at your own convenience?